src/main/resources/swagger.api/schemas/query.json

```diff
@@ -56,6 +56,10 @@
         "description": "Date and time when the query execution was completed",
         "type": "string",
         "format": "date-time"
+      },
+      "crossTenant": {
+        "description": "Indicates whether a query is cross-tenant",
+        "type": "boolean"
```

Comment:
Should we add an explicit default? Otherwise, I think the default will be `null`, which is unfortunate